src/status_im2/common/scroll_page/view.cljs

```diff
@@ -36,7 +36,7 @@
    top-nav
    title-colum
    navigate-back?]
-  (let [input-range         (if platform/ios? [-47 10] [0 92])
+  (let [input-range         (if platform/ios? [-47 10] [0 10])
```

Comment:
changes to this component also affect the chat messages pages too, right? Can you also verify that all pages using this component are not affected by this adjustment.

Reply:
this component is not currently being used in messages only discover community and  community home screen. The animation looks much better across all screens al though they may need some improvement especially when messages implements this functionality